raresGabriel01/home-assignment
# [ISSUE-5] Storybook setup

# Description
Added a storybook setup in order to better visualize the components.
Each component will have its own entry in the storybook with auto-generated docs.
Added a dummy `t-grid` for testing purposes.

# Issue link(s)
#5 

# How to test?

1. Open a terminal
2. `cd` into the root of the project
3. run `ng run home-assignment:storybook`
4. the browser should open with the storybook page
5. Go through the entries and see if the dummy `t-grid` is available.

# Attachments 
![image](https://github.com/raresGabriel01/home-assignment/assets/56756497/c5bc10fe-7363-4137-a3a2-246136c88384)
